test/android/core/javatests/src/org/xwalk/core/xwview/test/AddandRemoveJavascriptInterfaceTest.java

```diff
@@ -53,17 +63,25 @@ private void raisesExceptionAndSetTitle(String script) throws Throwable {
     }
 
     @SmallTest
-    @Feature({"AddJavascriptInterface"})
-    public void testAddJavascriptInterface() throws Throwable {
+    @Feature({"AddandRemoveJavascriptInterface"})
```

Comment:
"add" and "and" are separate words, so you need to capitalize this string correctly ("AddAndRemove", not "AddandRemove").